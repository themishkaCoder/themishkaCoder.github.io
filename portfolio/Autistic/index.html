<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Autistic</title>
    <link rel="stylesheet" href="style.css">
    <link rel="stylesheet" href="normalize.css">
</head>
<body>
    <header>
            
        <div class="wrapper">
            <div class="menu-background">
                <div class="menu">
                    <h1>Autistic</h1>
                    <button>Service</button>
                    <button>About</button>
                    <button>Contact</button>
                </div>
            </div>

            <div class="header-content">
                <h2>Lorem ipsum dolor sit amet <br>
                        consectetur adipiscing</h2>
                <input type="text" placeholder="search">         
            </div>
        </div>
    </header>

    <article>
        <div class="article-content">
        <form action="">
            <!-- <input type="text" placeholder="Select destinanion">
            <input type="text" placeholder="From">
            <input type="text" placeholder="To">
            <input type="text" placeholder="Guests"> -->
            
            <div class="inputs">
                <input type="text" placeholder="Select destinanion">
                <input type="text" placeholder="From">
                <input type="text" placeholder="To">
                
            </div>
            <input type="text" placeholder="Guests">
            <button>Check Availability</button>
        </form>

        <div class="galereya">
            <div class="small-img">
                <a href=""><img src="images/1.jpg" alt=""></a>
                <div class="hover-fon">
                    <h2>Click to open</h2>
                </div>
                
            </div>
            <div class="small-img">
                <a href=""><img src="images/2.jpg" alt=""></a>
                <div class="hover-fon">
                    <h2>Click to open</h2>
                </div>
            </div>

            <div class="small-img">
                <a href=""><img src="images/3.jpg" alt=""></a>
                <div class="hover-fon">
                    <h2>Click to open</h2>
                </div>
                <!-- <h2>Click to open</h2> -->
            </div>

            <div class="small-img">
                <a href=""><img src="images/4.jpg" alt=""></a>
                <div class="hover-fon">
                    <h2>Click to open</h2>
                </div>
            </div>

            <div class="big-img">
                <a href=""><img src="images/5.jpg" alt=""></a>
                <div class="hover-fon-big">
                    <h2>Click to open</h2>
                </div>
            </div>
        </div>
        </div>
    </article>

    <div class="houses-menu">
        <h2>New Houses</h2>
        <div class="adverts">
            <div class="advert">
                <div class="advert-img"><a href=""><img src="images/7.jpg" alt=""></a></div>
                <div class="advert-p">
                    <p class="advert-main-p">
                        Lorem ipsum dolor sit <br>
                        amet consectetur adipiscing
                    </p>
                    <p class="advert-submain-p">
                        Lorem: 1, Ipsum: 4
                    </p>
                    <p class="advert-price">
                        $599
                    </p>
                </div>
            </div>

            <div class="advert">
                <div class="advert-img"><a href=""><img src="images/7.jpg" alt=""></a></div>
                <div class="advert-p">
                    <p class="advert-main-p">
                        Lorem ipsum dolor sit <br>
                        amet consectetur adipiscing
                    </p>
                    <p class="advert-submain-p">
                        Lorem: 1, Ipsum: 4
                    </p>
                    <p class="advert-price">
                        $599
                    </p>
                </div>
            </div>

            <div class="advert">
                <div class="advert-img"><a href=""><img src="images/7.jpg" alt=""></a></div>
                <div class="advert-p">
                    <p class="advert-main-p">
                        Lorem ipsum dolor sit <br>
                        amet consectetur adipiscing
                    </p>
                    <p class="advert-submain-p">
                        Lorem: 1, Ipsum: 4
                    </p>
                    <p class="advert-price">
                        $599
                    </p>
                </div>
            </div>

            <div class="advert">
                <div class="advert-img"><a href=""><img src="images/7.jpg" alt=""></a></div>
                <div class="advert-p">
                    <p class="advert-main-p">
                        Lorem ipsum dolor sit <br>
                        amet consectetur adipiscing
                    </p>
                    <p class="advert-submain-p">
                        Lorem: 1, Ipsum: 4
                    </p>
                    <p class="advert-price">
                        $599
                    </p>
                </div>
            </div>

            <div class="advert">
                <div class="advert-img"><a href=""><img src="images/7.jpg" alt=""></a></div>
                <div class="advert-p">
                    <p class="advert-main-p">
                        Lorem ipsum dolor sit <br>
                        amet consectetur adipiscing
                    </p>
                    <p class="advert-submain-p">
                        Lorem: 1, Ipsum: 4
                    </p>
                    <p class="advert-price">
                        $599
                    </p>
                </div>
            </div>

            <div class="advert">
                <div class="advert-img"><a href=""><img src="images/7.jpg" alt=""></a></div>
                <div class="advert-p">
                    <p class="advert-main-p">
                        Lorem ipsum dolor sit <br>
                        amet consectetur adipiscing
                    </p>
                    <p class="advert-submain-p">
                        Lorem: 1, Ipsum: 4
                    </p>
                    <p class="advert-price">
                        $599
                    </p>
                </div>
            </div>

            <div class="advert">
                <div class="advert-img"><a href=""><img src="images/7.jpg" alt=""></a></div>
                <div class="advert-p">
                    <p class="advert-main-p">
                        Lorem ipsum dolor sit <br>
                        amet consectetur adipiscing
                    </p>
                    <p class="advert-submain-p">
                        Lorem: 1, Ipsum: 4
                    </p>
                    <p class="advert-price">
                        $599
                    </p>
                </div>
            </div>

            <div class="advert">
                <div class="advert-img"><a href=""><img src="images/7.jpg" alt=""></a></div>
                <div class="advert-p">
                    <p class="advert-main-p">
                        Lorem ipsum dolor sit <br>
                        amet consectetur adipiscing
                    </p>
                    <p class="advert-submain-p">
                        Lorem: 1, Ipsum: 4
                    </p>
                    <p class="advert-price">
                        $599
                    </p>
                </div>
            </div>
        </div>
        <button>See more</button>
    </div>

    <div class="second-galereya">
        <h2>NEWEST FROM BLOG</h2>
        <div class="images">
            <div class="img">
                <a href=""><img src="images/6.jpg" alt=""></a>
                <div class="second-galereya-p">
                    <p class="second-galereya-main-p">
                        Lorem ipsum dolor sit <br>
                        amet consectetur adipiscing
                    </p>

                    <p class="second-galereya-date">04/01/2017</p>
                
                </div>
            </div>
            <div class="img">
                <a href=""><img src="images/7.jpg" alt=""></a>
                <div class="second-galereya-p">
                    <p class="second-galereya-main-p">
                        Lorem ipsum dolor sit <br>
                        amet consectetur adipiscing
                    </p>
    
                    <p class="second-galereya-date">04/01/2017</p>
                    
                </div>
            </div>

            <div class="img">
                <a href=""><img src="images/8.jpg" alt=""></a>
                <div class="second-galereya-p">
                    <p class="second-galereya-main-p">
                        Lorem ipsum dolor sit <br>
                        amet consectetur adipiscing
                    </p>
    
                    <p class="second-galereya-date">04/01/2017</p>
                </div>
            </div>
        </div>
        <button>See more</button>
    </div>

    <div class="contact">
        <div class="container">
            <h2>CONTACT US</h2>
            <form action="">
                <input type="text" placeholder="Your name">
                <input type="text" placeholder="Phone number">
                <input type="text" placeholder="Type your message...">
                <!-- <textarea name="letter" id="" cols="10" rows="10" placeholder="Type your message...">
                    Type your message...
                </textarea> -->
                <button>SUBMIT</button>
            </form>
        </div>
    </div>

    <footer>
        <div class="footer-wrapper">
            <div class="footer-menu">
                <p class="footer-main-p">
                    Menu
                </p>
                <a href="">Service</a>
                <a href="">About</a>
                <a href="">Contact</a>
            </div>

            <div class="know-more">
                <p class="footer-main-p">
                    Know more
                </p>
                <a href="">Terms</a>
                <a href="">Affiliate</a>
                <a href="">Pricing</a>
                <a href="">Career</a>
            </div>

            <div class="partners">
                <p class="footer-main-p">
                    Partners
                </p>
                <a href="">Airnb</a>
                <a href="">TripAdvisor</a>
            </div>

            <div class="contacts">
                <p class="footer-main-p">
                    Contacts
                </p>
                <p class="footer-contact-p">
                    +8 (800) 000-00-00 <br>
                    mail@example.com
                </p>
            </div>

            <div class="social">
                <div class="facebook"><a href=""></a></div>
                <div class="twitter"><a href=""></a></div>
                <div class="youtube"><a href=""></a></div>
            </div>

            
        </div>
            <div class="end">
                <hr>
                <div class="item">
                    <p class="end-p">
                        Autistic 2017
                    </p>

                    <p class="end-link">
                        www.bisenbaev.com
                    </p>
                </div>
            </div>
    </footer>
</body>
</html>