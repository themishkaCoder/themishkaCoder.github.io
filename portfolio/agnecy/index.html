<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Agnecy</title>
    <link rel="stylesheet" href="css/style.css">
</head>
<body>
    
    <div class="container">
        <div class="navbar">
            <ul>
                <li><a href="">HOME</a></li>
                <li><a href="">PORTFOLIO</a></li>
                <li><a href="">ABOUT US</a></li>
                <li><a href="">CONTACT</a></li>
            </ul>
        </div>
    </div>

</body>
</html>