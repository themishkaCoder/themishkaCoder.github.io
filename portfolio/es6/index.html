<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>ES6</title>
</head>
<body>
    <div class="bez">

    </div>


    <div class="iz">

    </div>

    <p class="test"></p>
    <p class="test"></p>
    <p class="test"></p>
    <p class="test"></p>
    <p class="test"></p>



    <script>
        var es = "es6";
        var a = "Без " + es + ".";
        var bez = document.getElementById("bez");
        var iz = document.getElementById("iz");
        var array = document.getElementsByClassName("test");

        // bez.innerText= a;

        const template = `
            <h2>С использыванием ${es} </h2>
        `;

        
        //document.body.innerHTML = template + "\n" + a;

        for(let item of array){
            item.innerText = "Главная";
        }

        function helloName(name){
            console.log(`<h1>Hello my name is ${name}</h1>`);
            // return `<h1>Hello my name is ${name}</h1>`
        }
        helloName("Jhon");
    </script>
</body>
</html>