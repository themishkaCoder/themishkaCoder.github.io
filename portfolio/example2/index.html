<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Document</title>
    <link rel="stylesheet" href="style.css">
    <link rel="stylesheet" href="normalize.css">
</head>
<body>
   
    <header>
      <div class="content">
      
      
      <div class="rl">
       <div class="left-header-title">
        
           FreebiesGallery
        
       </div>
       
       <div class="right-header-title">
        Go to website
       </div>
          </div>
       
       <div class="center-content">
           <p class="centP">PROFESSIONAL WEBSITES</p>
           
       </div>
       
       <div class="center-content">
           <p class="centP2">IN JUST $299</p>
       </div>
       
       <div class="header-img">
           <img src="images/macbook.png" alt="">
       </div>
       
      </div>
    </header>
    
    <article>
        
        <p class="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam ante urna, placerat eu euismod dictum, aliquam et sapien. Ut at risus commodo, eleifend sem cursus, interdum velit. </p><br><br>
        
        <div class="btm">
        <button class="website"><p>Create web site now</p></button>
        </div><br><br>
        
        <div class="plusu">
        
         <div class="text">
            
             <p class="main">Easy Setup Wizard</p>
            
             <p class="submain">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam ante urna, placerat eu euismod dictum, aliquam et sapien. </p>
             
             <p class="main">Full Brand Control</p>
            
             <p class="submain">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam ante urna, placerat eu euismod dictum, aliquam et sapien. </p>
            
         </div>
         
         <div class="img">
         
         <img src="macinterface.jpg" alt="">
         
         </div>
         
         
         
         <div class="img">
         
         <img src="macinterface.jpg" alt="">
         
         </div>
         
         <div class="text">
            
             <p class="main">Photo Gallery</p>
            
             <p class="submain">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam ante urna, placerat eu euismod dictum, aliquam et sapien. </p>
             
             <p class="main">Menu Editor</p>
            
             <p class="submain">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam ante urna, placerat eu euismod dictum, aliquam et sapien. </p>
            
         </div>
         
        
        </div>
        
    </article>
    
    <div class="greenline">
        
        <div class="centere">
            
            <div class="text2">
            <p class="thirtypix">QUICK & EASY!</p>
            <p class="pp">Lorem ipsum dolor sit amet, consectetur adipiscing elit. </p>
            </div>
            
            <div class="button">
            
            
            <button class="bt2">SIGN UP NOW</button>
            
            
            
            </div>
            
        </div>
        
    </div>
    
    <div class="slider">
        
        <p class="bold">There are 29,364 Happy Website Owners!</p>
        <p class="description">
            “Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam ante urna, placerat eu euismod dictum, aliquam et sapien. Ut at risus commodo, eleifend sem cursus, interdum velit.”
        </p>
        
    </div>
    
    <footer>
        
        <div class="center2">
            
            <div class="text3">
                
                <p class="second-description">
                    Don't want to receive anymore emails? Unsubscribe
Freebiesgallery, 329 Notingham Street, San Francisco, CA 94307
© Copyright 2013 FreebiesGallery. All rights reserved.

                </p>
                
            </div>
            
            <!--------------------------------->
            
            <div class="social">
                
                <a href="#"><img src="images/Facebook.png" alt=""></a>
                <a href="#"><img src="images/twitter.png" alt=""></a>
                <a href="#"><img src="images/google-plus.png" alt=""></a>
                <a href="#"><img src="images/LinkedIn.png" alt=""></a>
                <a href="#"><img src="images/Pinterest.png" alt=""></a>
                
            </div>
            
            
        </div>
        
    </footer>
</body>
</html>