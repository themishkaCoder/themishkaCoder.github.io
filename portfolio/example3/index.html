<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Landing page</title>
    <link rel="stylesheet" href="style.css">
    <link rel="stylesheet" href="normalize.css">
</head>
<body>
    <header>
        <div class="wrapper">
            <div class="menu">
                <button class="menu-button">Home</button>
                <button class="menu-button">Portfolio</button>
                <button class="menu-button">About us</button>
                <button class="menu-button">Contact</button>
            </div>

            <div class="header-content">
                <div class="header-text">
                    <h1>
                        OUR STRONG<br> ORGANISATION
                    </h1>
                    <p>Lorem ipsum dolor sit amet,
                    consectetur adipis cing elit,
                    sed do eiusmod tempor incididunt
                    ut labore et. ectetur adig ipis cing elit,
                        sed do eiusmod tempor incididunt.</p>
                    <button>Contact us</button>
                </div>
                <div class="video">
                <iframe width="550" height="360" src="https://www.youtube.com/embed/qXDkolsFhms" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>  
                </div>  
            </div>
        </div>
    </header>

    <article></article>

    <footer></footer>
</body>
</html>