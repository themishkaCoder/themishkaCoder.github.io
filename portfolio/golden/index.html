<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Golden</title>
    
    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/css/bootstrap.min.css" integrity="sha384-GJzZqFGwb1QTTN6wy59ffF1BuGJpLSa9DkKMp0DgiMDm4iYMj70gZWKYbI706tWS" crossorigin="anonymous">
    <link rel="stylesheet" href="style.css">
    <link rel="stylesheet" href="css/hover.css">
    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
</head>
<body>
    
    <header>
        <div class="container">
            <div class="menu">
                <nav class="navbar navbar-expand-lg">
                    <div class="navbar-collapse" id="navbarSupportedContent">
                        <div class="container">
                            <div class="row d-flex justify-content-between">
                                <div class="col">
                                    <a class="navbar-brand" href="#"><img src="images/golden.png" alt=""></a>
                                </div>
                                <div class="col d-flex justify-content-end">
                                    <ul id="menu" class="navbar-nav">

                                        <button id="menuButton" class="d-lg-none" onclick="openMenu()"><i class="fas fa-bars"></i></button>

                                        <li class="d-none d-lg-block nav-item">
                                          <a class="nav-link" href="#">Home</a>
                                        </li>
                                        <li class="d-none d-lg-block nav-item">
                                            <a class="nav-link" href="#">Service</a>
                                        </li>
                                                        
                                        <li class="d-none d-lg-block nav-item">
                                            <a class="nav-link" href="#">Portfolio</a>
                                        </li>
                        
                                        <li class="d-none d-lg-block nav-item">
                                            <a class="nav-link" href="#">About</a>
                                        </li>
                        
                                        <li class="d-none d-lg-block nav-item">
                                          <a class="nav-link" href="#">Contact</a>
                                        </li>
                                        
                                        
                                    </ul>
                                </div>
                            </div>
                        </div>
                    </div>
                </nav>
            </div>

            <div class="welcome text-center">
                <div class="row">
                    <div class="col">
                        <h2>Welcome To Our Studio!</h2>
                        <h1>IT'S NICE TO MEET YOU</h1>
                    </div>
                </div>
                
                <div class="row">
                    <div class="col">
                        <button>TELL ME MORE</button>
                    </div>
                </div>
            </div>
        </div>
        
    </header>

    <article>
      <div class="container">
        <div class="row">
          <div class="col text-center">
            <h2 class="bold-text">SERVICES</h2>
            <p>Proin iaculis purus consequat sem cure</p>
          </div>
        </div>

        <div class="service">
          <div class="row">

            <div class="col-md-12 col-lg-4 text-center">
              <img src="images/1.png" alt="">
              <h3>E-Commerce</h3>
              <p>Proin iaculis purus consequat sem cure 
                digni ssim. Donec porttitora entum suscipit 
                aenean rhoncus posuere odio in tincidunt.
              </p>
            </div>

            <div class="col-md-12 col-lg-4 text-center">
              <img src="images/1.png" alt="">
              <h3>E-Commerce</h3>
              <p>Proin iaculis purus consequat sem cure 
                digni ssim. Donec porttitora entum suscipit 
                aenean rhoncus posuere odio in tincidunt.
              </p>
            </div>

            <div class="col-md-12 col-lg-4 text-center">
              <img src="images/1.png" alt="">
              <h3>E-Commerce</h3>
              <p>Proin iaculis purus consequat sem cure 
                digni ssim. Donec porttitora entum suscipit 
                aenean rhoncus posuere odio in tincidunt.
              </p>
            </div>


          </div>
        </div>
      </div>
    </article>

    <section class="portfolio">
        <div class="container">
          <div class="row">
            <div class="col text-center">
              <h2 class="bold-text">OUR PORTFOLIO</h2>
              <p>Proin iaculis purus consequat sem cure.</p>
            </div>
          </div>

          <div class="glr-photos">
            <div class="row">
              <div class="col-12 col-md-6 col-lg-4 text-center d-flex justify-content-center">
                <div class="glr-content hvr-float-shadow">
                  <div class="glr-img hvr-grow-rotate"><img src="images/port/1.jpg" alt=""></div>
                  <div class="glr-text">
                    <h4>Ebony & Ivory</h4>
                    <p>Branding</p>
                  </div>
                </div>
              </div>

              <div class="col-12 col-md-6 col-lg-4 text-center text-center d-flex justify-content-center">
                <div class="glr-content hvr-float-shadow">
                  <div class="glr-img hvr-grow-rotate"><img src="images/port/1.jpg" alt=""></div>
                  <div class="glr-text">
                    <h4>Ebony & Ivory</h4>
                    <p>Branding</p>
                  </div>
                </div>
              </div>

              <div class="col-12 col-lg-4 text-center text-center d-flex justify-content-center">
                <div class="glr-content hvr-float-shadow">
                  <div class="glr-img hvr-grow-rotate"><img src="images/port/1.jpg" alt=""></div>
                  <div class="glr-text">
                    <h4>Ebony & Ivory</h4>
                    <p>Branding</p>
                  </div>
                </div>
              </div>

              

              <div class="col-12 col-md-6 col-lg-4 text-center d-flex justify-content-center">
                <div class="glr-content hvr-float-shadow">
                  <div class="glr-img hvr-grow-rotate"><img src="images/port/1.jpg" alt=""></div>
                  <div class="glr-text">
                    <h4>Ebony & Ivory</h4>
                    <p>Branding</p>
                  </div>
                </div>
              </div>

              <div class="col-12 col-md-6 col-lg-4 text-center text-center d-flex justify-content-center">
                <div class="glr-content hvr-float-shadow">
                  <div class="glr-img hvr-grow-rotate"><img src="images/port/1.jpg" alt=""></div>
                  <div class="glr-text">
                    <h4>Ebony & Ivory</h4>
                    <p>Branding</p>
                  </div>
                </div>
              </div>

              <div class="col-12 col-lg-4 text-center text-center d-flex justify-content-center">
                <div class="glr-content hvr-float-shadow">
                  <div class="glr-img hvr-grow-rotate"><img src="images/port/1.jpg" alt=""></div>
                  <div class="glr-text">
                    <h4>Ebony & Ivory</h4>
                    <p>Branding</p>
                  </div>
                </div>
              </div>
            </div>
          </div>

          <!-- <div class="glr-photos">
              <div class="row">
                <div class="col-4 text-center d-flex justify-content-end">
                  <div class="glr-content">
                    <div class="glr-img"><img src="images/port/1.jpg" alt=""></div>
                    <div class="glr-text">
                      <h4>Ebony & Ivory</h4>
                      <p>Branding</p>
                    </div>
                  </div>
                </div>
  
                <div class="col-4 text-center text-center d-flex justify-content-center">
                  <div class="glr-content">
                    <div class="glr-img"><img src="images/port/1.jpg" alt=""></div>
                    <div class="glr-text">
                      <h4>Ebony & Ivory</h4>
                      <p>Branding</p>
                    </div>
                  </div>
                </div>
  
                <div class="col-4 text-center text-center d-flex justify-content-start">
                  <div class="glr-content">
                    <div class="glr-img"><img src="images/port/1.jpg" alt=""></div>
                    <div class="glr-text">
                      <h4>Ebony & Ivory</h4>
                      <p>Branding</p>
                    </div>
                  </div>
                </div>
              </div>
            </div> -->
        </div>
    </section>



    <section class="about-us">
      <div class="container">
        <div class="row">
          <div class="col text-center">
            <h2>About Us</h2>
          </div>
        </div>

        <div class="row">
          <div class="col d-none d-xl-block">
            <div class="text">
                <h3>July 2010 <br>
                Our Humble Beginnings
            </h3>
            <p>
                Proin iaculis purus consequat sem cure 
                digni ssim. Donec porttitora entum suscipit 
                aenean rhoncus posuere odio in tincidunt. Proin 
                iaculis purus consequat sem cure digni 
                ssim. Donec porttitora entum suscipit.
            </p>
            </div>
          </div>

          <div class="col d-flex justify-content-center align-items-end">
            <div class="time-img hvr-grow-rotate">
              <img src="images/timeline/1.jpg" alt="" class="">
            </div>
            
          </div>

          <div class="col">
            <div class="text-start d-lg-block d-xl-none">
                <h3>July 2010 <br>
                Our Humble Beginnings
            </h3>
            <p>
                Proin iaculis purus consequat sem cure 
                digni ssim. Donec porttitora entum suscipit 
                aenean rhoncus posuere odio in tincidunt. Proin 
                iaculis purus consequat sem cure digni 
                ssim. Donec porttitora entum suscipit.
            </p>
            </div>
          </div>
        </div>


        <div class="row">
          <div class="col d-none d-xl-block">
            
          </div>

          <div class="col d-flex justify-content-center align-items-end">
            <div class="time-img hvr-grow-rotate">
              <img src="images/timeline/1.jpg" alt="" class="">
            </div>
            
          </div>

          <div class="col">
            <div class="text-start">
                <h3>July 2010 <br>
                Our Humble Beginnings
            </h3>
            <p>
                Proin iaculis purus consequat sem cure 
                digni ssim. Donec porttitora entum suscipit 
                aenean rhoncus posuere odio in tincidunt. Proin 
                iaculis purus consequat sem cure digni 
                ssim. Donec porttitora entum suscipit.
            </p>
            </div>
          </div>
        </div>



        <div class="row">
          <div class="col d-none d-xl-block">
            <div class="text">
                <h3>July 2010 <br>
                Our Humble Beginnings
            </h3>
            <p>
                Proin iaculis purus consequat sem cure 
                digni ssim. Donec porttitora entum suscipit 
                aenean rhoncus posuere odio in tincidunt. Proin 
                iaculis purus consequat sem cure digni 
                ssim. Donec porttitora entum suscipit.
            </p>
            </div>
          </div>

          <div class="col d-flex justify-content-center align-items-end">
            <div class="time-img hvr-grow-rotate">
              <img src="images/timeline/1.jpg" alt="" class="">
            </div>
            
          </div>

          <div class="col">
            <div class="text-start d-lg-block d-xl-none">
                <h3>July 2010 <br>
                Our Humble Beginnings
            </h3>
            <p>
                Proin iaculis purus consequat sem cure 
                digni ssim. Donec porttitora entum suscipit 
                aenean rhoncus posuere odio in tincidunt. Proin 
                iaculis purus consequat sem cure digni 
                ssim. Donec porttitora entum suscipit.
            </p>
            </div>
          </div>
        </div>



        <div class="row">
          <div class="col d-none d-xl-block">
            
          </div>

          <div class="col d-flex justify-content-center align-items-end">
            <div class="time-img hvr-grow-rotate">
              <img src="images/timeline/1.jpg" alt="" class="">
            </div>
            
          </div>

          <div class="col">
            <div class="text-start">
                <h3>July 2010 <br>
                Our Humble Beginnings
            </h3>
            <p>
                Proin iaculis purus consequat sem cure 
                digni ssim. Donec porttitora entum suscipit 
                aenean rhoncus posuere odio in tincidunt. Proin 
                iaculis purus consequat sem cure digni 
                ssim. Donec porttitora entum suscipit.
            </p>
            </div>
          </div>
        </div>
      </div>
    </section>


    <section class="team">
      <div class="container">
        <div class="row">
          <div class="col text-center">
            <h2 class="bold-text">OUR AMAZING TEAM</h2>
            <p>Proin iaculis purus consequat sem cure.</p>
          </div>
        </div>

        <div class="team-content">
          <div class="row">
            <div class="col d-flex flex-column justify-content-center">
              <div class="team-photo m-auto hvr-wobble-skew"><img src="" alt=""></div>
              <div class="team-text text-center">
                <h5>Kimberly Thompson</h5>
                <p>Marketer</p>
              </div>
              <div class="team-social d-flex flex-row justify-content-center">
                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-bounce-in">
                    <i class="fab fa-twitter"></i>
                  </div>
                </a>
                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-bounce-in">
                    <i class="fab fa-facebook-f"></i>
                  </div>
                </a>
                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-bounce-in">
                    <i class="fab fa-pinterest-p"></i>
                  </div>
                </a>
              </div>
            </div>



            <div class="col d-flex flex-column justify-content-center">
              <div class="team-photo m-auto hvr-wobble-skew"><img src="" alt=""></div>
              <div class="team-text text-center">
                <h5>Kimberly Thompson</h5>
                <p>Marketer</p>
              </div>
              <div class="team-social d-flex flex-row justify-content-center">
                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-bounce-in">
                    <i class="fab fa-twitter"></i>
                  </div>
                </a>
                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-bounce-in">
                    <i class="fab fa-facebook-f"></i>
                  </div>
                </a>
                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-bounce-in">
                    <i class="fab fa-pinterest-p"></i>
                  </div>
                </a>
              </div>
            </div>


            <div class="col d-flex flex-column justify-content-center">
              <div class="team-photo m-auto hvr-wobble-skew"><img src="" alt=""></div>
              <div class="team-text text-center m-auto">
                <h5>Kimberly Thompson</h5>
                <p>Marketer</p>
              </div>
              <div class="team-social d-flex flex-row justify-content-center">
                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-bounce-in">
                    <i class="fab fa-twitter"></i>
                  </div>
                </a>
                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-bounce-in">
                    <i class="fab fa-facebook-f"></i>
                  </div>
                </a>
                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-bounce-in">
                    <i class="fab fa-pinterest-p"></i>
                  </div>
                </a>
              </div>
            </div>
          </div>
        </div>

        <div class="row">
          <div class="col text-center">
            <p>Proin iaculis purus consequat sem cure  digni ssim donec porttitora entum suscipit <br>  
                aenean rhoncus posuere odio in tincidunt proin iaculis.</p>
          </div>
        </div>
      </div>
    </section>

    <section class="logos">
      <div class="container">
        <div class="row">
          <div class="col-12 col-sm-6 col-lg d-flex justify-content-center align-items-center mb-4">
            <img src="images/len.png" alt="">
          </div>

          <div class="col-12 col-sm-6 col-lg d-flex justify-content-center align-items-center mb-4">
            <img src="images/wordpress.png" alt="">
          </div>

          <div class="col-12 col-sm-6 col-lg d-flex justify-content-center align-items-center mb-4">
            <img src="images/tuts.png" alt="">
          </div>

          <div class="col-12 col-sm-6 col-lg d-flex justify-content-center align-items-center mb-4">
            <img src="images/micro.png" alt="">
          </div>
        </div>
      </div>
    </section>


    <section class="contact-us">
      <div class="container">
        <div class="row">
          <div class="col text-center">
            <div class="contact-text">
              <h2>CONTACT US</h2>
              <p>Proin iaculis purus consequat sem cure.</p>
            </div>
          </div>
        </div>

        <div class="contact">
          <div class="row">
            <div class="col-6">
                <input type="text" placeholder="YOUR NAME*"> 
                <input type="text" placeholder="YOUR E-MAIL*">
                <input type="text" placeholder="SUBJECT">
              </div>
        
              <div class="col-6">
                <textarea name="" id="" cols="30" rows="10" placeholder="YOUR MESSAGE*"></textarea>
              </div>
          </div>

          <div class="row">
            <div class="col text-center">
              <button>SEND MESSAGE</button>
            </div>
          </div>
        </div>
        
      </div>
    </section>


    <footer>
      <div class="container">
        <div class="row">
          <div class="col-4">
            <p>© Copyright 2014 FreebiesXpress.com</p>
          </div>

          <div class="col-8">
            <div class="footer-social d-flex flex-row justify-content-center">
                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-buzz-out">
                    <i class="fab fa-twitter"></i>
                  </div>
                </a>
                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-buzz-out">
                    <i class="fab fa-facebook-f"></i>
                  </div>
                </a>
                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-buzz-out">
                    <i class="fab fa-pinterest-p"></i>
                  </div>
                </a>

                <a href="">
                  <div class="social-icon d-flex justify-content-center align-items-center hvr-buzz-out">
                      <i class="fab fa-google-plus-g"></i>
                  </div>
                </a>
              </div>
          </div>
        </div>
      </div>
    </footer>


    <!-- Optional JavaScript -->
    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
    <script src="js/main.js"></script>
    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.6/umd/popper.min.js" integrity="sha384-wHAiFfRlMFy6i5SRaxvfOCifBUQy1xHdJ/yoi7FRNXMRBu5WHdZYu1hA6ZOblgut" crossorigin="anonymous"></script>
    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.2.1/js/bootstrap.min.js" integrity="sha384-B0UglyR+jN6CkvvICOB2joaf5I4l3gm9GU6Hc1og6Ls7i6U/mkkaduKaBhlAXv9k" crossorigin="anonymous"></script>
</body>
</html>